databases/s3/database.go

```diff
@@ -303,21 +306,29 @@ func (db *Database) s3ObjectToDescriptor(key string) (map[string]any, error) {
 		return nil, fmt.Errorf("error retrieving metadata for S3 object %s: %v", key, err)
 	}
 
+	mediatype := aws.ToString(headOutput.ContentType)
+	if mediatype == "" {
+		mediatype = "application/octet-stream"
+	}
```

Comment:
@jeff-cohere - the iRODS S3 proxy doesn't seem to always return a media type, and if it's left blank the validation of the metadata fails. I wasn't sure what do in these cases